bin/templates/project/__PROJECT_NAME__/config.xml

```diff
@@ -65,6 +65,10 @@
     <feature name="LocalStorage">
         <param name="ios-package" value="CDVLocalStorage"/>
     </feature>
+    <feature name="Console">
```

Comment:
oh is that a cordova-cli integration requirement or something? I was testing it without using cordova-cli, just by creating a project via `./bin/create`, etc.